orca/variables.tf

```diff
@@ -0,0 +1,185 @@
+## Required
+
+### Module
+
+variable "DEPLOY_NAME" {
+  type = string
+}
+
+### Orca
+
+variable "orca_db_user_password" {
+  description = "Password for RDS Orca database user authentication"
+  type        = string
+}
+
+variable "orca_dlq_subscription_email" {
+  type        = string
+  description = "The email to notify users when messages are received in dead letter SQS queue due to orca restore failure."
+  default = "test@email.com" ## TODO: Set this via secret and remove default
+}
+
+variable "orca_default_bucket" {
+  type        = string
+  description = "Default ORCA S3 Glacier bucket to use."
+}
+
+variable "orca_reports_bucket" {
+  type        = string
+  description = "ORCA Reports Bucket"
+}
+
+variable "orca_buckets" {
+  type = list(string)
+}
+
+## Optional
+
+### Module
+
+variable "MATURITY" {
+  type    = string
+  default = "dev"
+}
+
+variable "elb_account_id" {
+  type = string
+  default = "797873946194"
```

Comment:
I don't think an account_id should have a default value.  Am I missing something?